### ISSUE TITLE
[enhancement]: Add support for E330 professional cameras

### ISSUE THREAD

**Issue by ibmaster** [homebridge-eufy-security/plugin#424]
### Describe the solution you'd like

Hello,

I have E330 wired 24/7 4K cameras associated with my HomeBase 3, however it seems the plugin does not support these cameras. Please consider supporting them in future plugin releases. Thank you.

**Comment by lenusch** [comment]
Yes please. I tried hours until find out, not really supported. 
Any ETA? thx

**Comment by lenoxys** [comment]
it's been back ported in 2.3.6 hksv

**Comment by lenusch** [comment]
> it's been back ported in 2.3.6 hksv

Hi thank you. I installed that Version and now it at least appeasrs in Homebridge and Homekit after manuel insert. But nothing working. I don't have Settings at homebridge (as with HB2 + Cams) and i cant see Picture on Homekit. It sais "no answer, this camera is not responding" (self translated).. 

And its only the S220 SoloCam which is showing up (but not work as described). 
The 24/7 Camera E330 it not working at all. Shows not up under Devices in Homebridge. But in Config its there, but nearly no Options. I also set unbridge for both cams.

**Comment by tranceme** [comment]
Thanks @lenusch  for connecting me to this thread. 

I tried this version (Beta 2.3.7) and no luck with HB3 and/or E330 Cam  When I go to add it manually, it seems at though the HB3 is bridge. I don't think I can add the E330 Cam on its own. 

I tried in the Plug In Settings using Unbridge Camera and then add it manually using homebridge instance (e.g. 123-45-678).  It adds the bridge still and then sees the camera as device.  I assume that is incorrect and it should just add camera. Not completely sure how the unbridge/bridge mode works.  But, when I added the camera, it just shows up in Home app as unsupported.

**Comment by lenusch** [comment]
Yes its still not working

**Comment by DrWhy44** [comment]
happy to see this being worked on, thanks folks.

**Comment by lenusch** [comment]
I have still problems. I even can't watch both cams on Homekit, its kinda disturbing each other. And then i think to watch that nothing works anymore.

**Comment by captcook32** [comment]
Wanted to check in to see if this was still being worked on. I have the same issue and would love support for these cameras.

**Comment by lenusch** [comment]
ITS WORKIN FOR ME :-) 
https://github.com/homebridge-eufy-security/plugin/issues/470#issuecomment-2074225255

**Comment by lenoxys** [comment]
**Thank you for taking the time to report this issue.**

As part of a repository cleanup, we are closing issues that have been open for more than 12 months without resolution. A **beta version** of the plugin has been released that includes significant changes and improvements.

We encourage you to try the latest beta and see if your issue has been addressed. If the problem persists, please feel free to open a new issue with updated details and logs from the latest version.

Thank you for your patience and support!
